src/main/kotlin/uk/gov/dluhc/notificationsapi/config/NotifyClientConfiguration.kt

```diff
@@ -139,6 +182,12 @@ class OverseasNotifyEmailTemplateConfiguration(
     sourceType: SourceType = SourceType.OVERSEAS,
     receivedEnglish: String,
     receivedWelsh: String,
+    rejectedParentGuardianEnglish: String?,
```

Comment:
On the Infra API we've also added "required" comms for the parent guardian proof/qualifying address/documents - not sure if we need them adding here too?

Reply:
Yep that's going to be the next ticket I start